src/settings.js

```diff
@@ -1635,6 +1635,10 @@ var FETCH = 0;
 // [link]
 var ASMFS = 0;
 
+// If set to 1, uses new filesystem implementation
```

Comment:
Maybe add some notes here, like
* Experimental for now, try at your own risk.
* Intended to eventually replace most of the current JS filesystem code.

Reply:
This makes sense